packages/client/ui/react-ui/src/hooks/useCrossmint.tsx

```diff
@@ -35,14 +39,21 @@ export function CrossmintProvider({
         }
     }, []);
 
+    const setRefreshToken = useCallback((refreshToken: string | undefined) => {
```

Comment:
Although I see us already using useCallback throughout this file, so this comment should apply to all cases or to none, I see it as an overhead using useCallback here. 

The purpose of useCallback is to prevent a function being created on each render, making it more performant. However, I don't see that criteria applying for a function that performs a simple if check.

But this is just something that took my attention, it's not a comment to change anything

Reply:
Yeah, you're right. I can remove in this file at least